Gemfile

```diff
@@ -32,6 +32,9 @@ gem 'jquery-rails'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+# a library for generating fake data.
+gem 'faker'
```

Comment:
If so, should you limit it with `development` and` test`? We do not need it for `staging` or` production`.